.travis.yml

```diff
@@ -73,8 +73,10 @@ jobs:
                  cmake --build /root/project/parts/multipass/build --target covreport'
       after_success:
       - bash <(curl -s https://codecov.io/bash) || echo "Codecov did not collect coverage reports"
-    - if: (type = pull_request AND head_repo = CanonicalLtd/multipass)
-          OR (type != pull_request AND repo = CanonicalLtd/multipass)
+    - if: |
+      repo = CanonicalLtd/multipass AND
+             ((type = pull_request AND head_repo = repo)
+              OR type != pull_request)
```

Comment:
Good to factor out the repo, but I think I would rather read 

```
    - if: repo = CanonicalLtd/multipass AND
          (type != pull_request OR head_repo = repo)
```

Reply:
You're totally right, I got wound up in these conditions :)

Reply:
Fixed.